lib/reek/cli/command/todo_list_command.rb

```diff
@@ -36,13 +36,13 @@ def scan_for_smells
         end
 
         def groups_for(smells)
-          @groups ||= begin
-            Hash[
-              smells.group_by(&:smell_type).map do |smell_type, smells_for_type|
-                [smell_type, { 'exclude' => smells_for_type.map(&:context) }]
+          @groups ||=
+            begin
+              todos = smells.group_by(&:smell_class).map do |smell_class, smells_for_class|
```

Comment:
I like the change to `smell_class`, don't get me wrong. But shouldn't we then kill `SmellDetector.smell_type` throughout Reek?